docker-compose.yml

```diff
@@ -32,6 +32,9 @@ services:
       - MYSQL_DATABASE=app_db
       - MYSQL_USER=app_user
       - MYSQL_PASSWORD=1234
+      - MYSQL_ENGINE=mysql.connector.django
```

Comment:
The `MYSQL_ENGINE` environment variable is not a standard MySQL configuration variable. If this is intended for a specific application configuration, ensure it is necessary. Otherwise, consider removing it.